core/src/integration/java/com/radixdlt/integration/steady_state/deterministic/rev2/consensus_ledger_sync/MultiNodeRebootTest.java

```diff
@@ -215,11 +214,15 @@ private DeterministicTest createTest(
                         numValidators,
                         Decimal.of(1),
                         GenesisConsensusManagerConfig.Builder.testWithRoundsPerEpoch(
-                            this.roundsPerEpoch)),
+                                this.roundsPerEpoch)
+                            .totalEmissionXrdPerEpoch(Decimal.of(0))),
                     REv2StateManagerModule.DatabaseType.ROCKS_DB,
+                    new DatabaseFlags(true, false),
                     StateComputerConfig.REV2ProposerConfig.transactionGenerator(
-                        new REV2TransactionGenerator(), 1)),
-                SyncRelayConfig.of(5000, 10, 5000L))));
+                        new REV2TransactionGenerator(), 1),
+                    false,
+                    true),
+                SyncRelayConfig.of(100, 10, 500))));
```

Comment:
Can we have a comment explaining all of these parameters and how they're important?

I'm worried some of this (eg avoiding stake changing over time) is quite fragile, so any hints to future people regarding where to look if this becomes flaky in future I think would be useful.